tests/unix/mod_os.py

```diff
@@ -0,0 +1,20 @@
+# This module is not entirely compatible with CPython
+import os
+
+
+os.putenv("TEST_VARIABLE", "TEST_VALUE")
+
+print(os.getenv("TEST_VARIABLE"))
+print(os.getenv("TEST_VARIABLE", "TEST_DEFAULT_VALUE"))
+
+os.unsetenv("TEST_VARIABLE")
+
+print(os.getenv("TEST_VARIABLE"))
+print(os.getenv("TEST_VARIABLE", "TEST_DEFAULT_VALUE"))
+
+print(os.system("true"))
+
+print(os.urandom(4).hex())
```

Comment:
Rather than regex'ing the output (which could be anything, so nothing really gained from testing the output), test the length and/or type of the return value.  Then this doesn't need to be a special test (and simplicity is always better!).

Reply:
great idea! done